openmc/mgxs_library.py

```diff
@@ -99,47 +88,45 @@ class XSdata(object):
     fission : dict of numpy.ndarray
         Group-wise fission cross section.
     kappa_fission : dict of numpy.ndarray
-        Group-wise kappa-fission cross section.
+        Group-wise kappa_fission cross section.
     chi : dict of numpy.ndarray
         Group-wise fission spectra ordered by increasing group index (i.e.,
         fast to thermal). This attribute should be used if making the common
         approximation that the fission spectra does not depend on incoming
         energy. If the user does not wish to make this approximation, then
         this should not be provided and this information included in the
-        :attr:`XSdata.nu_fission` attribute instead.
+        ``nu_fission`` attribute instead.
+    chi_prompt : dict of numpy.ndarray
+        Group-wise prompt fission spectra ordered by increasing group index
+        (i.e., fast to thermal). This attribute should be used if chi from
+        prompt and delayed neutrons is being set separately.
+    chi_delayed : dict of numpy.ndarray
+        Group-wise delayed fission spectra ordered by increasing group index
+        (i.e., fast to thermal). This attribute should be used if chi from
+        prompt and delayed neutrons is being set separately.
     nu_fission : dict of numpy.ndarray
-        Group-wise fission production cross section vector (i.e., if
-        :attr:`XSdata.chi` is provided), or is the group-wise fission
-        production matrix.
-    inverse_velocities : dict of numpy.ndarray
-        Inverse of velocities, in units of sec/cm.
-
-    Notes
-    -----
-    The parameters containing cross section data have dimensionalities which
-    depend upon the value of :attr:`XSdata.representation` as well as the
-    number of Legendre or other angular dimensions as described by
-    :attr:`XSdata.order`. The :attr:`XSdata.vector_shape`,
-    :attr:`XSdata.matrix_shape`, and :attr:`XSdata.pn_matrix_shape` properties
-    are provided to obtain the dimensionality of the data for each temperature.
-
-    The following are cross sections which should use each of these properties:
-
-    vector_shape :
-        total, absorption, fission, kappa_fission, chi, nu_fission (if chi is
-        provided), inverse_velocities
-    matrix_shape :
-        multiplicity_matrix, nu_fission (if chi is not provided)
-    pn_matrix_shape :
-        scatter_matrix
+        Group-wise fission production cross section vector (i.e., if ``chi`` is
+        provided), or is the group-wise fission production matrix.
+    prompt_nu_fission : dict of numpy.ndarray
```

Comment:
With this PR, yes. I have another PR with the code all written and ready to go which has a `prompt_nu_fission matrix` and `delayed_nu_fission matrix` that can replace the respective `chi`s. To keep the PRs digestible, I decided to break them up. I'm happy to add them to this PR if that would make things easier.